pkg/controllers/resources/nodes/syncer.go

```diff
@@ -307,14 +307,6 @@ func (s *nodeSyncer) Sync(ctx *synccontext.SyncContext, event *synccontext.SyncE
 
 	s.translateUpdateBackwards(event.Host, event.Virtual)
 
-	// Set the marker of managed-by vcluster so that
```

Comment:
This is now done in the `translate.SyncHostMetadataToVirtual(...)` func (that the nodes syncer is calling).